src/SonarScanner.MSBuild.Common/Telemetry/TelemetryUtils.cs

```diff
@@ -52,6 +52,15 @@ public static void AddTelemetry(ILogger logger, HostInfo serverInfo)
         logger.AddTelemetryMessage("dotnetenterprise.s4net.serverInfo.serverUrl", serverUrl);
     }
 
+    public static void AddTelemetry(ILogger logger, BuildSettings settings)
```

Comment:
Will we need more telemetry from BuildSettings?
If not I will remove this method and call `AddTelemetryMessage` directly.

Reply:
removed it - we can add a method if needed.